build.gradle

```diff
@@ -25,7 +25,7 @@ plugins {
 group 'org.opensearch.driver'
 
 // keep version in sync with version in Driver source
-version '2.0.0.0'
+version '1.0.0.0'
```

Comment:
change to 1.0.0.0?

Reply:
latest release is 1.1.0.1, this should be 1.2.0.0?

Reply:
yes - this actually should be in sync with java/org/opensearch/jdbc/internal/Version.java:line 12 
`  // keep this in sync with the gradle version
    Current(1, 0, 0, 0);
`

Reply:
1.2.0.0 it is...